### PR TITLE
improve turret descriptions

### PR DESCRIPTION
**Pull request by ccondry**
#### Summary

SUMMARY: Content "fix a few typos, add a few missing words, and remove double whitespace after periods."

#### Purpose of change

This is to improve readability of the turret descriptions.

#### Testing

Verified I only made text changes, and validated the JSON with jsonlint.com

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Double spaces after periods are a convention started much earlier in the project. It is now enforced for consistency, as all strings with 2+ sentences will have double spaces.

File: data/json/monsters/turrets.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "id": "mon_laserturret",
     "type": "MONSTER",
     "name": { "str": "laser turret" },
-    "description": "The TX-5LR Cerberus is an upgrade to its predecessors.  It features a state of the art revolving laser cannon system with three barrels that charge from solar cells embedded in its hull.",
+    "description": "The TX-5LR Cerberus is an upgrade to its predecessors. It features a state of the art revolving laser cannon system with three barrels that charge from solar cells embedded in its hull.",
```
Comment (olanti-p) [inline data/json/monsters/turrets.json:34]:
```suggestion
    "description": "The TX-5LR Cerberus is an upgrade to its predecessors.  It features a state of the art revolving laser cannon system with three barrels that charge from solar cells embedded in its hull.",
```

File: data/json/monsters/turrets.json [hunk 2]
```diff
@@ -69,7 +69,7 @@
...
+    "description": "These TALON-derived riot control platforms were widely publicized a few years before the cataclysm as a new semi-autonomous device that could fire less-lethal rounds with far more accuracy than a human, ensuring safer hits against a target's limbs. They were quickly adopted by prisons and inner city police forces, where they demonstrated that 'less lethal' is not the same as 'non-lethal'. In the days before the cataclysm, massive stockrooms of the things were put into circulation. On the bright side, although it shoots autonomously, it requires a human operator to relocate, so it's not so mobile anymore.",
```
Comment (olanti-p) [inline data/json/monsters/turrets.json:72]:
```suggestion
    "description": "These TALON-derived riot control platforms were widely publicized a few years before the cataclysm as a new semi-autonomous device that could fire less-lethal rounds with far more accuracy than a human, ensuring safer hits against a target's limbs.  They were quickly adopted by prisons and inner city police forces, where they demonstrated that 'less lethal' is not the same as 'non-lethal'.  In the days before the cataclysm, massive stockrooms of the things were put into circulation.  On the bright side, although it shoots autonomously, it requires a human operator to relocate, so it's not so mobile anymore.",
```

File: data/json/monsters/turrets.json [hunk 3]
```diff
@@ -121,7 +121,7 @@
     "id": "mon_turret_light",
     "type": "MONSTER",
     "name": { "str": "light turret" },
-    "description": "The General Atomics TX-1 Guardian, an popular automated turret equipped with a 9mm carbine.  Initially it was designed mainly for corporate property protection and police needs, however due its popularity it ended up being used even for military needs.  Hight quality targetting system allows it to engage targets even at medium range.",
+    "description": "The General Atomics TX-1 Guardian, an popular automated turret equipped with a 9mm carbine. Initially it was designed mainly for corporate property protection and police needs, however due to its popularity, it ended up being used even for military needs. Its high-quality targeting system allows it to engage targets even at medium range.",
```
Comment (olanti-p) [inline data/json/monsters/turrets.json:124]:
```suggestion
    "description": "The General Atomics TX-1 Guardian, an popular automated turret equipped with a 9mm carbine.  Initially it was designed mainly for corporate property protection and police needs, however due to its popularity, it ended up being used even for military needs.  Its high-quality targeting system allows it to engage targets even at medium range.",
```

File: data/json/monsters/turrets.json [hunk 4]
```diff
@@ -183,7 +183,7 @@
     "id": "mon_turret_medium",
     "type": "MONSTER",
     "name": { "str": "medium milspec turret" },
-    "description": "The General Atomics TX-5 Sentinel, an automated milspec turret equipped with a 5.56mm gun.  Direct upgare of orignal TX-1 Guardian for military use.  Designed to efficiently take down enemy personnel protected by body armor at medium range.",
+    "description": "The General Atomics TX-5 Sentinel, an automated milspec turret equipped with a 5.56mm gun. A direct upgrade of orignal TX-1 Guardian for military use. Designed to efficiently take down enemy personnel protected by body armor at medium range.",
```
Comment (olanti-p) [inline data/json/monsters/turrets.json:186]:
```suggestion
    "description": "The General Atomics TX-5 Sentinel, an automated milspec turret equipped with a 5.56mm gun.  A direct upgrade of orignal TX-1 Guardian for military use.  Designed to efficiently take down enemy personnel protected by body armor at medium range.",
```

File: data/json/monsters/turrets.json [hunk 5]
```diff
@@ -245,7 +245,7 @@
     "id": "mon_turret_longrange",
     "type": "MONSTER",
     "name": { "str": "longrange milspec turret" },
-    "description": "The General Atomics TX-7 Sniper, an automated milspec turret equipped with a 7.62mm gun.  Designed as counter sniper turret to cover vulnerability of TX-5 Sentinel to sniper fire.  Its advanced target recognition system and 7.62mm gun allows it to engage targets protected by modern body armor at very long range.",
+    "description": "The General Atomics TX-7 Sniper, an automated milspec turret equipped with a 7.62mm gun. Designed as a counter-sniper turret to cover the vulnerability of TX-5 Sentinel to sniper fire. Its advanced target recognition system and 7.62mm gun allows it to engage targets protected by modern body armor at very long range.",
```
Comment (olanti-p) [inline data/json/monsters/turrets.json:248]:
```suggestion
    "description": "The General Atomics TX-7 Sniper, an automated milspec turret equipped with a 7.62mm gun.  Designed as a counter-sniper turret to cover the vulnerability of TX-5 Sentinel to sniper fire.  Its advanced target recognition system and 7.62mm gun allows it to engage targets protected by modern body armor at very long range.",
```

**Review by olanti-p — APPROVED** [review]

**Pull request merged**